docs/development/workflow/get_devel_version.rst

```diff
@@ -216,36 +216,43 @@ and you should be rewarded with::
 "Activate" the development version of astropy
 ---------------------------------------------
 
-Right now you have the development version of `Astropy`_, but python will not
+Right now you have the development version of astropy, but python will not
 see it. Though there are more sophisticated ways of managing multiple versions
-of `Astropy`_, for now this straightforward way will work (if you want to jump
+of astropy, for now this straightforward way will work (if you want to jump
 ahead to the more sophisticated method look at :ref:`virtual_envs`).
 
 .. note::
-    There are a couple of circumstances in which this quick method of
-    activating your copy of `Astropy`_ will NOT work and you need to go
-    straight to using a virtual python environment:
+    If you want to work on C or Cython code in `Astropy`_, this quick method
+    of activating your copy of astropy will *not* work _ -- you need to go 
+    straight to using a virtual python environment.
 
-    + You use Python 3.
-    + You want to work on C or Cython code in `Astropy`_.
+If you have decided to use the recommended "activation" method with
+``pip``, please note the following: Before trying to install, 
+check that you have the required dependencies: "cython" and "jinja2". 
+If not, install them with ``pip``. Note that on some platforms, 
+the pip command is ``pip3`` instead of ``pip``, so be sure to use 
+this instead in the examples below if that is the case. 
+If you have any problem with different versions of ``pip`` installed,
+try aliasing to resolve the issue. If you are unsure about which ``pip`` 
+version you are using, try the command ``which pip`` on the terminal.
 
-In the directory where your copy of `Astropy`_ is type::
+In the directory where your copy of astropy is type::
 
-    pip install -e .[all,test]
+    pip install -e .
```

Comment:
if I see correctly the ``[all,test]`` part is never added back to elsewhere in the docs, but that can be done in a follow-up.